playbooks/adhoc/uninstall.yml

```diff
@@ -55,6 +55,10 @@
     - pcsd
     failed_when: false
 
+  - shell: "atomic uninstall atomic-openshift-master"
+    changed_when: False
```

Comment:
I've used the existing pattern that is already used for Docker containers.  It looks like uninstall.yml is trying to do "remove as much as possible and never fail":

For example, this is done with Docker containers:
```
  - shell: docker rm -f "{{ item }}"-master "{{ item }}"-node
    changed_when: False
    failed_when: False
    with_items:
    - openshift-enterprise
    - atomic-enterprise
    - origin
```